src/app/components/ATIAnalytics/params/index.test.ts

```diff
@@ -245,20 +255,32 @@ describe('ATIAnalytics params', () => {
 
     it('should return the correct PGL url', () => {
       const url = buildATIUrl(
-        PGL,
         { ...requestContext, pageType: PHOTO_GALLERY_PAGE },
         serviceContext,
+        PGL,
       );
       expect(url).toMatchInlineSnapshot(
         `"s=598285&s2=atiAnalyticsProducerId&p=pageIdentifier&r=0x0x24x24&re=1024x768&hl=00-00-00&lng=en-US&x1=[urn%3Abbc%3Acps%3A4d36f80b-8711-0b4e-8da0-ef76ae8ac470]&x2=[responsive]&x3=[atiAnalyticsAppName]&x4=[language]&x5=[http%253A%252F%252Flocalhost%252F]&x7=[article-photo-gallery]&x8=[simorgh]&x9=[headline%2520-%2520brandName]&x11=[1970-01-01T00%3A00%3A00.000Z]&x12=[1970-01-01T00%3A00%3A00.000Z]"`,
       );
     });
 
+    it('should return the correct Homepage url', () => {
+      const url = buildATIUrl(
+        { ...requestContext, pageType: HOME_PAGE },
+        serviceContext,
+        undefined,
+        homePage,
+      );
+      expect(url).toMatchInlineSnapshot(
+        `"s=598285&s2=atiAnalyticsProducerId&p=kyrgyz.page&r=0x0x24x24&re=1024x768&hl=00-00-00&lng=en-US&x1=[urn%3Abbc%3Atipo%3Atopic%3Acm7682qz7v1t]&x2=[responsive]&x3=[atiAnalyticsAppName]&x4=[pcm]&x5=[http%253A%252F%252Flocalhost%252F]&x7=[index-home]&x8=[simorgh]&x9=[pageTitle]"`,
+      );
```

Comment:
I would love to make this snapshot a little bit more readable (and use as an example when we refactor it later). 

Is there a way in which we could use URLSearchParams to parse these params nicely?

Sometime along the lines of... 

```
expect(new URLSearchParams(url)).toMatchInlineSnapshot();
```

I would expect to see a more readable object as the snapshot. Here's an example of when I console logged it:

```
params: URLSearchParams {
    's' => '598285',
    's2' => 'atiAnalyticsProducerId',
    'p' => 'kyrgyz.page',
    'r' => '0x0x24x24',
    're' => '1024x768',
    'hl' => '00-00-00',
    'lng' => 'en-US',
    'x1' => '[urn:bbc:tipo:topic:cm7682qz7v1t]',
    'x2' => '[responsive]',
    'x3' => '[atiAnalyticsAppName]',
    'x4' => '[pcm]',
    'x5' => '[http%3A%2F%2Flocalhost%2F]',
    'x7' => '[index-home]',
    'x8' => '[simorgh]',
    'x9' => '[pageTitle]' }
}
```

Reply:
We're applying updates in: https://github.com/bbc/simorgh/pull/10891 

We had an approach which I did using snapshots:

```
   it('should return the correct Homepage url', () => {
      const url = buildATIUrl(
        { ...requestContext, pageType: HOME_PAGE },
        serviceContext,
        undefined,
        homePage,
      );

      const queryParams = new URLSearchParams(url)
        .toString()
        .replace(/&/g, '\n&')
        .trim();

      const expectedSnapshot = `
"s=598285
&s2=atiAnalyticsProducerId
&p=kyrgyz.page
&r=0x0x24x24
&re=1024x768
&hl=00-00-00
&lng=en-US
&x1=${encodeURIComponent('[urn:bbc:tipo:topic:cm7682qz7v1t]')}
&x2=${encodeURIComponent('[responsive]')}
&x3=${encodeURIComponent('[atiAnalyticsAppName]')}
&x4=${encodeURIComponent('[pcm]')}
&x5=${encodeURIComponent('http://localhost/')}
&x7=${encodeURIComponent('[index-home]')}
&x8=${encodeURIComponent('[simorgh]')}
&x9=${encodeURIComponent('[pageTitle]')}"
      `.trim();

      expect(queryParams).toMatchInlineSnapshot(expectedSnapshot);
    });
```

However, the indentation issues would not be compatible with linting and the encoding associated with `/` was something we couldn't get around. Have opted for a solution that's based on equality of value than simply just a snapshot. That change has now been committed to the branch above!